sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -270,11 +256,19 @@ def get_named_nested_pvalues(pvalueish):
     tagged_values = pvalueish.items()
   else:
     if isinstance(pvalueish, (pvalue.PValue, pvalue.DoOutputsTuple)):
-      yield None, pvalueish
+      # For transforms that only have a tagged PCollection as an output,
+      # propagate that tag forward.
+      if first_iteration and isinstance(pvalueish, pvalue.PValue):
+        yield pvalueish.tag, pvalueish
```

Comment:
I think this may break some google3 runners. Can you ensure that this imports correctly? (Could you also explain why this is needed?)

Reply:
Yep, removed.